include/envoy/grpc/context.h

```diff
@@ -78,6 +80,8 @@ class Context {
    */
   virtual void chargeResponseMessageStat(const Upstream::ClusterInfo& cluster,
                                          const RequestNames& request_names, uint64_t amount) PURE;
+
+  virtual GoogleGrpcStatNames& googleGrpcStatNames() PURE;
```

Comment:
i'll add a doxygen comment here...just want to see if there are other comments first before pushing another change that kicks off CI.